src/esp/bindings/PhysicsObjectBindings.cpp

```diff
@@ -398,6 +443,13 @@ void declareArticulatedObjectWrapper(py::module& m,
                              ("Get a dict mapping Habitat object ids to this " +
                               objType + "'s link ids.")
                                  .c_str())
+      .def("get_link_id_from_name",
+           &ManagedArticulatedObject::getLinkIdFromName,
+           ("Get this " + objType +
+            "'s articulated link id specified by the passed "
+            "link_name.")
```

Comment:
What happens when the link name is invalid? Error/crash or null return?

Reply:
ESP_CHECK failure.